lib/Alchemy/Phrasea/Command/Record/RescanFilesMetadata.php

```diff
@@ -0,0 +1,302 @@
+            $sbasId = $databox->get_sbas_id();
+            $sbasName = $databox->get_dbname();
+
+            $field = $this->getField($sbasId, $destination);
+
+            if ($field == null) {
+                $output->writeln(sprintf("<error>Field %s not found on database %s </error>", $destination, $sbasName));
+                continue;
+            }
+
+            if ( $col != null) {
+                $collection = $this->getCollection($sbasId, $col);
+                if ($collection == null) {
+                    $output->writeln(sprintf("<error>collection %s not found on database %s </error>", $col, $sbasName));
+                    continue;
+                }
+
+                $collId = $collection->get_coll_id();
+
+                $clauses[] = "coll_id = " . $collId;
+            }
+
+            if ($source == null) {
+                // use the destination field src
+                $sourceTag = $field->get_original_source();
+            }
+
+            $metaStructId = $field->get_id();
+            $fieldName = $field->get_name();
+
+            $action = "set";
+
+            if ($field->is_multi()) {
+                $action = 'add';
+
+                // define what value to replace ??
+//                if ($multi == 'replace') {
+//                    $action = 'replace';
+//                }
+            }
+
+            $output->writeln(sprintf("<comment>Working on database %s with Id : %d</comment>", $sbasName, $sbasId));
+
+
+            $sql_where = join(" AND ", $clauses);
+
+            $sql = "SELECT * FROM record WHERE " . $sql_where;
+
+            $stmt = $databox->get_connection()->prepare($sql);
+
+            if ($dry) {
+                $output->writeln("<info>" . $sql . "</info>");
+            }
+
+            $stmt->execute();
+            $rows = $stmt->fetchAll(\PDO::FETCH_ASSOC);
+
+            foreach ($rows as $row) {
+                $record = $databox->get_record($row['record_id']);
+
+                // skip non empty field if overwrite is not set
+                if ($overwrite == null && !$this->isEmptyField($record, $fieldName)) {
+                    continue;
+                }
+
+                try {
+                    $results = $record->getFileMetadataByTag($sourceTag);
+                } catch (\Exception $e) {
+                    $output->writeln("<error> can not get metadata into the document file : " . $e->getMessage() . "</error>");
+                    $results = [];
+                }
+
+                if ($results != NULL) {
+                    $metadatas = [];
+                    foreach ($results as $result) {
+                        $metadatas[] = [
+                            'action'         => $action,
+                            'meta_struct_id' => $metaStructId,
+                            'meta_id'        => null,
+                            'value'          => $result
+                        ];
+                    }
```

Comment:
for multi-v, it is also possible to use value = [a,b,c] : use "results" directly in a single action.